meta-luneos/recipes-qt/qt5/qtlocation-luneos-plugin_git.bb

```diff
@@ -16,7 +16,7 @@ SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE};"
 S = "${WORKDIR}/git"
 
 FILES_${PN} += " \
-    ${libdir}/plugins \
+    ${libdir}/qt5/plugins/position \
```

Comment:
This is wrong, use the qmake5_paths bbclass and proper variable.

Reply:
${OE_QMAKE_PATH_PLUGINS}/postition

Reply:
Updated :) Adjusted PV as well while we're at it.